bundle/quay-operator.subscription.yaml

```diff
@@ -1,11 +1,11 @@
 apiVersion: operators.coreos.com/v1alpha1
 kind: Subscription
 metadata:
-  name: quay-tng
+  name: quay-operator-tng
 spec:
-  channel: alpha
+  channel: preview-3.6
```

Comment:
I would leave this version agnostic unless we want to update this with each release. Could we somehow pull this in from the CSV as an annotation? Otherwise LGTM

Reply:
Good point! I changed this back to alpha and changed the upstream annotations to match.